deployment/ansible/group_vars/all

```diff
@@ -19,4 +19,8 @@ postgresql_support_libpq_version: "9.3.*"
 postgresql_support_repository_channel: "9.3"
 postgresql_version: "9.3"
 
+otp_repo: "https://github.com/azavea/OpenTripPlanner.git"
+otp_version: "77072a5b548e1777b23127ebedfcf08f2461c171"
+otp_jarfile: "./target/otp-0.13.0.jar"
```

Comment:
Could also just use a wildcard instead, like `otp-*.jar`.  I think that's what the `otp` script does.